packages/db/README.md

```diff
@@ -33,7 +33,8 @@ Add the following to your `truffle-config.js` file in order to enable Truffle DB
 
 ```
 db: {
-  enabled: true
+  enabled: true,
+  saveLocally: true // Optional - save the .db directory to the project directory. Default: false.
```

Comment:
This shouldn't be in this example anymore, since we've moved this option from truffle-config over to the user-level config